### ISSUE TITLE
Cannot import MANAGER_VERSION_MINIMUM, MANAGER_VERSION_RECOMMENDED

### ISSUE THREAD

**Issue by varunbpatil** [OpenVPN/openvpn3-indicator#14]
Error in the latest version.

They are named BACKEND_VERSION_MINIMUM and BACKEND_VERSION_RECOMMENDED in about.py

**Comment by agassi88** [comment]
same issue on Ubuntu 22.04 after latest openvpn3-indicator update:
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/bin/openvpn3-indicator/__main__.py", line 25, in <module>
  File "/usr/bin/openvpn3-indicator/openvpn3_indicator/__init__.py", line 35, in main
  File "/usr/bin/openvpn3-indicator/openvpn3_indicator/application.py", line 45, in <module>
ImportError: cannot import name 'MANAGER_VERSION_MINIMUM' from 'openvpn3_indicator.about' (/usr/bin/openvpn3-indicator/openvpn3_indicator/about.py)
```

**Comment by grzegorz-gutowski** [comment]
Sorry for that, I've tried to quickly fix #13, but then I mixed up files again.
It should be fixed by the last commit.

**Comment by varunbpatil** [comment]
Confirmed fixed.
